Lib/test/test_os.py

```diff
@@ -374,7 +374,7 @@ def test_statvfs_attributes(self):
 
         # Use the constructor with a too-long tuple.
         try:
-            result2 = os.statvfs_result((0,1,2,3,4,5,6,7,8,9,10,11,12,13,14))
+            result2 = os.statvfs_result((0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15))
```

Comment:
I don’t understand this change at all. Does the behaviour of the original version change? It looks like this code won’t fail the test if the constructor succeeds anyway. Maybe better to use _TestCase.assertRaises_ (if this needs testing at all).

Reply:
I've increased the size of the test by one since I've changed the statvfs return size by one, so to keep the same difference.  Anyway I agree it makes no sense, and I will drop it.